mypy.ini

```diff
@@ -18,7 +20,35 @@ disallow_subclassing_any = False
 disallow_untyped_decorators = False
 disallow_any_generics = False
 disallow_untyped_calls = False
-disallow_incomplete_defs = False
-disallow_untyped_defs = False
+# disallow_incomplete_defs = False
+# disallow_untyped_defs = False
 no_implicit_reexport = False
 warn_return_any = False
+
+# Temporary excludes for files with with many tedious errors
```

Comment:
Should this really be put into trunk right now?

Reply:
converted to a to-do